apps/web-client/src/lib/db/cr-sqlite/books.ts

```diff
@@ -75,6 +75,39 @@ async function _upsertBook(db: DB, book: BookData) {
 	);
 }
 
+async function _getMultipleBookData(db: DB, ...isbns: string[]): Promise<Array<Required<BookData> & { updatedAt: Date | null }>> {
+	const placeholders = isbns.map((_, i) => `(?, ${i})`).join(",\n");
+	const query = `
+		WITH input_list(isbn, ord) AS (
```

Comment:
TIL about `WITH` in SQL. Nice! I always used a literal `IN` in those cases.

Reply:
Felt the same way I saw it for the first time, thanks @fadwamahmoud